bitnami/airflow/Chart.lock

```diff
@@ -9,4 +9,4 @@ dependencies:
   repository: https://charts.bitnami.com/bitnami
   version: 14.8.11
 digest: sha256:41449a4ca6faecac5fe27c0abd9cc6c8cc495418b7ed92362659ea545f8f439d
-generated: "2021-09-02T22:01:47.392733506Z"
+generated: "2021-08-27T17:51:48.282848+02:00"
```

Comment:
You can safely remove the changes in the _Chart.lock_ from this PR, this file is going to be automatically updated by the test & release pipeline

Reply:
oks